### DISCUSSION TITLE
Use myria3d for classify UAV data

### DISCUSSION THREAD

**Discussion by BenoitURRUTY** [IGNF/myria3d#136] [Q&A]
Hi,

I recently started using Myria3D, and the test dataset is working well.

I’d like to use AI to classify UAV LiDAR datasets, but our UAV does not capture the infrared band. Is it possible to skip this part of the process? Alternatively, what kind of infrared dataset could we use to interpolate this data?

Thank you!

**Comment by leavauchier** [comment]
Hi @BenoitURRUTY, 
Thanks for your feedback!

The model that we have trained relies on infrared data to make its prediction, so I see 3 options:
- Keep the infrared band blank, in this case the predictions quality will drop (I guess it will mainly affect vegetation but it may have a bigger impact). One way to check if the quality is still good enough for your usage is to mimick missing infrared data on a dataset on which you have a ground truth
- Use external infrared data to interpolate the infrared value of the points, such as the IRC data from [BDORTHO](https://geoservices.ign.fr/bdortho) if your data data is in France. (or orthophotos from other sources). See [here](https://ignf.github.io/myria3d/tutorials/prepare_dataset.html) for help on how to add it to your data
- In case you have sufficient training UAV data, you can re-train the model on your data (see documetnation [here](https://ignf.github.io/myria3d/guides/train_new_model.html))

**Comment by BenoitURRUTY** [comment]
Thanks for this answer. I tried to put infrared at 0 with oblique data on cliff. It's seems to work pretty fine thanks. I will try the BDORTHO solution later 👍
